### PR TITLE
Text tweaks

### PR DESCRIPTION
**Pull request by stripedpajamas**
Addressed some of the low-hanging fruit from Eryn's feedback. Done in small commits so if anything isn't liked we can easily revert pieces of it. All the things I did I crossed out in her doc.

### CODE REVIEW

**Review by meowwwls — APPROVED** [review]

File: components/splash/cards/forDevelopersCards.js [hunk 1]
```diff
@@ -43,10 +43,12 @@ const Card = ({ icon, heading, text, showHowItWorksLink }) => (
         lineGap='1'
         lineColor='ocean'
       />
-      <Text>{text}</Text>
-      {showHowItWorksLink && (
-        <TextLink text='How does that work, exactly?' href='/how-it-works' />
-      )}
+      <Text>
+        {text}
```
Comment (joelwass) [inline components/splash/cards/forDevelopersCards.js:47]: wdyt about these changes? i liked the conversational voice

File: components/verify/verify_humanity.js [hunk 2]
```diff
@@ -82,17 +82,17 @@ const VerifyHumanity = () => {
...
         <UnderlinedHeading text='Verify' align='center' aria-hidden='true' />
         <Subheading>No robots allowed</Subheading>
-        <Text marginBottom='1rem'>
-          On behalf of the Open Source community, thank you for installing!
+        <Text marginBottom='0.5rem'>
+          On behalf of the Open Source community, thank you for signing up!
           &hearts;
         </Text>
         {!hasError && (
-          <Text fontWeight='bold' marginBottom='1.5rem' color='ocean'>
```
Comment (joelwass) [inline components/verify/verify_humanity.js:90]: why change the font weight and color?

File: pages/dashboard.js [hunk 3]
```diff
@@ -209,20 +207,19 @@ const Dashboard = () => {
...
                   <Heading
                     as='h3'
-                    fontSize='1.15rem'
-                    fontWeight='500'
+                    fontSize='0.9rem'
+                    fontWeight='normal'
                     id='user-session-count'
                   >
-                    Installs Performed
+                    Installations performed
```
Comment (joelwass) [inline pages/dashboard.js:222]: you and melissa have conflicting changes - both pushed at 1am lol. i like this wording though

File: components/common/header/authNav.js [hunk 4]
```diff
@@ -56,6 +56,7 @@ const AuthNav = ({ user, onLogout }) => {
               display={{ base: 'none', sm: 'initial' }}
               fontWeight='500'
               marginRight='1rem'
+              fontSize='0.80rem'
```
Comment (joelwass) [inline components/common/header/authNav.js]: text way too small revert, melissa can comment on why

File: components/dashboard/donationCard.js [hunk 5]
```diff
@@ -65,11 +64,13 @@ const DonationCard = ({ donationAmount, donationLoading, hasDonation, optOutOfAd
         )}
         <UnderlinedHeading
           as='h3'
+          fontWeight='normal'
+          textTransform='none'
           id='donation-amt'
-          text='Monthly Donation'
+          text='Donated monthly'
```
Comment (joelwass) [inline components/dashboard/donationCard.js]: don't like the past tense, doesn't imply this is an ongoing event - wdyt

File: components/dashboard/topTenPackagesView.js [hunk 6]
```diff
@@ -49,7 +49,6 @@ const TopTenChart = ({ topTenPackages }) => (
 const TopTenPackagesView = ({ topTenPackages }) => (
   <>
     <Flex flexDirection='row'>
-      <Icon name='duotoneStar' size='2em' marginRight='.5em' marginTop='-0.5em' />
```
Comment (joelwass) [inline components/dashboard/topTenPackagesView.js:52]: why :(

File: components/select/selectTierCards.js [hunk 7]
```diff
@@ -74,14 +74,15 @@ const TierCard = ({ onSelected, tier, onModalOpen }) => (
...
         padding='.25rem'
         textTransform='uppercase'
         position='absolute'
         left='0'
         top='0'
         width='100%'
-        letterSpacing='1px'
-        fontSize='.85rem'
+        letterSpacing='2px'
+        fontSize='.75rem'
```
Comment (joelwass) [inline components/select/selectTierCards.js]: bump this back up for a11y. also with this font weight i can barely see it!

File: components/user/settings/accountInformationSection.js [hunk 8]
```diff
@@ -91,7 +91,6 @@ const AccountInformationSection = ({ user }) => {
             lineHeight='1.2'
             transition='all 300ms ease-in-out'
           >
-            <Icon name='add' size='1rem' marginRight='1rem' />
```
Comment (joelwass) [inline components/user/settings/accountInformationSection.js:93]: i liked these icons. makes the site feel more alive

**Review by joelwass — CHANGES_REQUESTED** [review]

**Review by joelwass — APPROVED** [review]

**Pull request merged**